contracts/staking/contracts/src/immutable/MixinConstants.sol

```diff
@@ -43,4 +45,10 @@ contract MixinConstants is
     uint64 constant internal INITIAL_TIMELOCK_PERIOD = INITIAL_EPOCH;
 
     uint256 constant internal MIN_TOKEN_VALUE = 10**18;
+
+    // The address of the canonical WETH contract -- this will be used as an alternative to ether for paying protocol fees.
```

Comment:
Nit - please move this to `MixinDeploymentConstants` ~ which has constants that must be updated for each deployment.